extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -273,15 +273,22 @@ export class AmpLightboxGallery extends AMP.BaseElement {
    * Return a cleaned clone of the given element for building
    * carousel slides with.
    * @param {!Element} element
-   * @return {*} TODO(#23582): Specify return type
+   * @return {!Element}
    * @private
    */
   cloneLightboxableElement_(element) {
+    const fallback = element.getFallback();
```

Comment:
What about placeholders? Why can't we deep clone custom elements?

Reply:
I'm not sure that we can't, but we need to do some digging for why we originally didn't. The `deepClone` boolean explicitly checks that the target element is not an AMP element.

Reply:
I think it's because if we deep clone elements, they might behave weirdly when interacting with the runtime. I'm actually not sure what happens if we clone an initialized amp-element and there are various listeners registered to it. But you could try and see if it works.